README.md

```diff
@@ -58,9 +58,13 @@ python3 ./manim.py example_scenes.py SquareToCircle -pl
 
 ## Anaconda Install
 
-* Install sox and latex as above. 
-* Create a conda environment using `conda env create -f environment.yml`
-* **WINDOWS ONLY** Install `pyreadline` via `pip install pyreadline`. 
+* Install SoX and LaTeX as above.
+* Clone this repository as above: `git clone https://github.com/3b1b/manim.git`.
+* Navigate into the repository as above: `cd manim`.
+* Create a conda environment using `conda env create -f environment.yml`.
+* Activate the environment: `conda activate manim`.
+* Add the repo's directory to the Python path: `conda develop .`.
```

Comment:
This is useful if the user wants to write scripts outside of the `manim` root directory.